src/app/fdctl/run/run.h

```diff
@@ -6,33 +6,35 @@
 #include "../../../tango/xdp/fd_xsk.h"
 
 typedef struct {
-   int           pid;
-   char *        app_name;
-   char *        tile_name;
-   ulong         tile_idx;
-   ulong         idx;
-   uchar const * tile_pod;
-   uchar const * in_pod;
-   uchar const * out_pod;
-   fd_xsk_t    * xsk;
-   fd_xsk_t    * lo_xsk;
+   int                pid;
+   char *             app_name;
+   char *             tile_name;
+   ulong              tile_idx;
+   ulong              idx;
+   const uchar **     wksp_pod;
```

Comment:
How many workspaces are in this array? Would it be easier to make it a static array of 16 or something? I guess that would only save one alloca